src/main/java/codeu/controller/LoginServlet.java

```diff
@@ -74,7 +74,8 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     }
 
     if (!userStore.isUserRegistered(username)) {
-      User user = new User(UUID.randomUUID(), username, Instant.now());
+	//TO DO: Add a password. Currently null.
```

Comment:
At google we have a policy of writing the name of the person that wrote the TODO. So that in the future if someone doesn't know what the TODO means, we can take ask the person that wrote the TODO. Also nit instead of TO DO the coding guidelines at google say to just use it as one word.